crates/bevy_ui/Cargo.toml

```diff
@@ -31,7 +31,7 @@ bevy_window = { path = "../bevy_window", version = "0.14.0-dev" }
 bevy_utils = { path = "../bevy_utils", version = "0.14.0-dev" }
 
 # other
-taffy = { version = "0.4" }
+taffy = { git = "https://github.com/nicoburns/taffy", branch = "bevy-issue-13155" }
```

Comment:
I assume the plan is to revert this to 4.x once taffy releases with this changes?

Reply:
It will bump to 0.5 but yes, the plan is to revert to a crates.io (before this PR is merged).